src/components/SettingsDialog/ExportBoard/ExportHintHiddenColumns/ExportHintHiddenColumns.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/constants/style.scss";
+
+$icon-text--gap: 20px;
+$columns-list--padding-left: $icon-text--gap + 50px;
```

Comment:
This seems to me like a fault in the structure. Have you thought of using `display: grid` to have the list aligned with the text? So this "manual" movement to the right is not needed.

Reply:
I now reworked the layout using a `grid`, I'm happier with it than the previous design. The aforementioned variables are no longer required. What do you think?